test/main-test.js

```diff
@@ -77,11 +77,16 @@ describe('PercyClient', function() {
   });
 
   describe('createBuild', function() {
-    it('returns build data', function(done) {
-      let resources = [percyClient.makeResource({resourceUrl: '/foo%20bar', sha: 'fake-sha'})];
-      let commitData = percyClient.environment.commitData;
+    let resources, commitData, expectedRequestData, responseMock;
```

Comment:
define these variables within this `describe` scope to use them within the `beforeEach` block to tighten up the code a little bit.

Reply:
"Scope"/locality rules should take precedence here — if the variables are only used inside the beforeEach block and don't need to be "global" to the describe block then they should be declared in the beforeEach. Looks like `responseMock` and `resources` are reused, but the other 2 should probably just be declared in the block.